sdcm/nemesis.py

```diff
@@ -2081,8 +2082,9 @@ def check_ssl_reload_log(target_node, file_path, since_time):
                 raise LogContentNotFound('Reload SSL message not found in node log')
             return msg.stdout
 
-        ssl_files_location = '/etc/scylla/ssl_conf'
-        in_place_crt = self.target_node.remoter.run(f"cat {os.path.join(ssl_files_location, 'db.crt')}",
+        ssl_files_location = json.loads(
+            self.target_node.get_scylla_config_param("server_encryption_options"))["certificate"]
```

Comment:
if get_scylla_config_param will return 'None' this line will get exception. shouldn't we try to handle this case?